conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/Java8OptionalParamConverterProvider.java

```diff
@@ -69,7 +69,7 @@ public String toString(final T value) {
                     return new ParamConverter<T>() {
                         @Override
                         public T fromString(final String value) {
-                            return rawType.cast(Optional.ofNullable(value).map(s -> converter.fromString(value)));
+                            return rawType.cast(Optional.ofNullable(value).map(_s -> converter.fromString(value)));
```

Comment:
This is a fun one

Reply:
nice, this prevents the lambda from being optimized into a static lambda cache because it must capture the `value` reference